src/Components/Clock.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './Clock.scss';
+
+type Props = {
+  clockName: number,
+};
+
+type State = {
+  time: string;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    time: '',
+  };
+
+  timerId?: NodeJS.Timer;
+
+  componentDidMount() {
+    this.timerId = setInterval(() => {
+      const date: string = new Date().toLocaleTimeString();
+
+      this.setState({ time: date });
+
+      // eslint-disable-next-line
+      console.log(this.state.time);
+    }, 1000);
+  }
+
+  componentDidUpdate(prevClockName: Props) {
+    if (prevClockName.clockName !== this.props.clockName) {
+      // eslint-disable-next-line
+      console.log(` renamed from ${prevClockName.clockName} to ${this.props.clockName}`);
+    }
+  }
```

Comment:
When I hide clock time I still can change the clock name, but I don't see this message on the console

Reply:
@AlinaMatuschak  I tried to fix the problem, but I found another solution